app/views/sessions/new.html.erb

```diff
@@ -4,7 +4,7 @@
 	<div class="sign-in_mask"></div>
   <div class="sign-in_content">
   	<div class="sign-in_logo">Grace<span class="sign-in_logo-emphasized">Tunes</span></div>
-	  <%= link_to "/auth/google_oauth2", class: "sign-in_button", id: "sign_in" do %>
+	  <%= link_to "/auth/google_oauth2", method: :post, class: "sign-in_button", id: "sign_in" do %>
```

Comment:
Was this the main fix for the login bug?

Reply:
Yep. They got rid of the GET endpoint